clients/ts/signalr/src/ITransport.ts

```diff
@@ -2,9 +2,10 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 export enum HttpTransportType {
-    WebSockets,
-    ServerSentEvents,
-    LongPolling,
+    None = 0,
+    WebSockets = 1,
+    ServerSentEvents = 2,
+    LongPolling = 3,
```

Comment:
C# version:

    [Flags]
    public enum HttpTransportType
    {
        None = 0,
        WebSockets = 1,
        ServerSentEvents = 2,
        LongPolling = 4,
    }

Should LongPolling be 4 for consistency and to allow bitwise in the future?

Reply:
Yass